crawl-ref/source/directn.cc

```diff
@@ -825,6 +826,25 @@ void full_describe_view()
 }
 
 void do_look_around(const coord_def &whence)
+{
+    dist lmove = _look_around_target(whence);
+    if (lmove.isValid && lmove.isTarget && !lmove.isCancel
+        && !crawl_state.arena_suspended)
+    {
+        start_travel(lmove.target);
+    }
+}
+
+bool get_look_position(coord_def *c)
+{
+    dist lmove = _look_around_target(coord_def(0, 0));
```

Comment:
0,0 here makes it sound like it will put the player's cursor at the origin. What if you started at the player's position instead, and then subtracted the player's position afterwards? That makes it clearer that we're using player-centered coordinates.